attributes/default.rb

```diff
@@ -3,17 +3,20 @@
 
 case node['platform']
 when 'debian', 'ubuntu'
-  default['pgpool']['config']['package_name'] = 'pgpool2'
+  default['pgpool']['config']['package_name'] = 'pgpool-II-pg93' # Might override
```

Comment:
The centos package name should be fine, but the debian and ubuntu package name isn't valid and will fail install.